webapp/apps/btax/helpers.py

```diff
@@ -0,0 +1,262 @@
+from collections import namedtuple
+import numbers
+import os
+import pandas as pd
+import dropq
+import json
+import string
+import sys
+import time
+import six
+
+#Mock some module for imports because we can't fit them on Heroku slugs
+from mock import Mock
+import sys
+
+from ..taxbrain.helpers import (make_bool, convert_val,
+                                is_wildcard, int_to_nth,
+                                is_number, is_string,
+                                string_to_float, string_to_float_array,
+                                same_version, arrange_totals_by_row,
+                                round_gt_one_to_nearest_int, expand_1D,
+                                expand_2D, expand_list, leave_name_in,
+                                TaxCalcField)
+
+import btax
+from btax.util import read_from_egg
+MOCK_MODULES = ['numba', 'numba.jit', 'numba.vectorize', 'numba.guvectorize',
```

Comment:
we no longer need these mock modules because our Heroku slug size was increased to 600 MB.